numpy/_core/tests/_natype.py

```diff
@@ -178,7 +178,7 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
 
         if method != "__call__":
             raise ValueError(f"ufunc method '{method}' not supported for NA")
-        result = maybe_dispatch_ufunc_to_dunder_op(
+        result = maybe_dispatch_ufunc_to_dunder_op(  # noqa: F821
```

Comment:
I cannot find where `maybe_dispatch_ufunc_to_dunder_op` is supposed to be defined.